features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Permissions/Permission.jsx

```diff
@@ -124,13 +131,14 @@ class Permission extends Component {
                 const securitySchemes = this.createSecuritySchemeArray(this);
                 api_data.securityScheme = securitySchemes;
                 const promised_update = api.update(api_data);
+                const { intl } = this.props;
                 promised_update
                     .then((response) => {
-                        message.success('Permissions updated successfully');
+                        message.success(intl.formatMessage({ id: 'Apis.Details.Permissions.Permissions.permissions.updated.successfully', defaultMessage: 'Permissions updated successfully' }));
```

Comment:
Key seems not tallying with the standard we agreed. 
Since the file is Apis/Details/Permissions/Permission.jsx, key needs to be prefixed as Apis.Details.Permissions.Permission.<prefered_suffix>
Additional 's' in the key

Reply:
done